app/src/main/java/com/sun_asterisk/soundcloud_71/data/model/Song.java

```diff
@@ -0,0 +1,122 @@
+package com.sun_asterisk.soundcloud_71.data.model;
+
+public class Song {
+    private int mId;
+    private String mTitle;
+    private String mArtist;
+    private String mDuration;
+    private String mGenre;
+    private String mArtWorkUrl;
+    private String mStreamUrl;
+    private boolean mIsDownload;
```

Comment:
```suggestion
    private boolean isDownloadable;
```
Mà biến này k cần dùng đâu